File: solutions/typescript/VB_T2_010_sum_positives.ts
// VB-T2-010: Sum Positives -- TypeScript baseline
function sumPositives(arr: number[]): number { return arr.filter(x => x > 0).reduce((a, x) => a + x, 0); }
console.assert(sumPositives([-1,2,-3,4,0]) === 6);

File: solutions/python/VB_T2_010_sum_positives.py
"""VB-T2-010: Sum Positives -- Python baseline."""
def sum_positives(arr: list[int]) -> int:
    return sum(x for x in arr if x > 0)

if __name__ == "__main__":
    assert sum_positives([-1,2,-3,4,0]) == 6
    print("VB-T2-010 ok")
